packages/jest-preset-kyt-enzyme/jest-preset.js

```diff
@@ -10,9 +10,9 @@ const jestConfig = {
   setupFilesAfterEnv: [require.resolve('./setup')],
   // if projects spread this value, the import can be lost if it is not absolute
   snapshotSerializers: [require.resolve('enzyme-to-json/serializer')],
-  testMatch: ['**/*.test.js'],
+  testMatch: ['**/*.test.[jt]s?(x)'],
   testEnvironment: require.resolve('jest-environment-jsdom-global'),
-  collectCoverageFrom: ['**/*.js'],
+  collectCoverageFrom: ['**/*.{js,jsx,json,ts,tsx}'],
```

Comment:
[Jest's default `collectCoverageFrom` config](https://jestjs.io/docs/en/configuration.html#collectcoveragefrom-array) looks like this:

```json
{
  "collectCoverageFrom": [
    "**/*.{js,jsx}",
    "!**/node_modules/**",
    "!**/vendor/**"
  ]
}
```

As you can see, it uses this different kind of glob pattern, so I am following their lead here.